frontend/src/app/libs/common-components/components/input/input.component.ts

```diff
@@ -0,0 +1,85 @@
+import { ThrowStmt } from '@angular/compiler';
+import { AfterViewInit, ChangeDetectionStrategy, ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { ComponentTheme } from '../../shared/component-theme.enum';
+import { InputType } from '../../shared/input-type.enum';
+
+@Component({
+	selector: 'app-input',
+	templateUrl: './input.component.html',
+	styleUrls: ['./input.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class InputComponent {
+
+	@Input()
+	public hide: boolean = true;
+
+	@Input()
+	public inputType: string;
+
+	@Input()
+	public label: string = 'label';
+
+	@Input()
+	public placeholder: string = 'placeholder';
+
+	@Input()
+	public isRequired: boolean = true;
```

Comment:
just `required`

Reply:
also seems like we can remove it since ngForms provide `Validators.Required` by default